document authN scopes (#194)
temporalio/web: README.md

> ⚠️ This is currently a beta feature, [please report any and all issues to us!](https://github.com/temporalio/web/issues/new)

> Since v1.2, Temporal Web offers optional Oauth SSO authentication. You can enable it in 2 steps:

Reviewer: ```suggestion
Since v1.3, Temporal Web offers optional Oauth SSO authentication. You can enable it in 2 steps:
```

    Create config.yml file on your machine, for example at `~/Desktop/config.yml`. 
    Start the docker image, providing the path to your config.yml file using external volume flag (-v). Leave the path after the semicolon as is: 

    ```bash
    docker run --network host -v ~/Desktop/config.yml:/usr/app/server/config.yml temporalio/web:latest
    ```

    </details>

    In future, multiple Oauth providers may be supported, however for now we only read the first Oauth provider under the `providers` key above.

    Common Oauth Providers and their docs:

    - Google: https://developers.google.com/identity/protocols/oauth2/openid-connect
    - Auth0: https://auth0.com/docs/protocols/configure-okta-as-oauth2-identity-provider
    - Okta: https://developer.okta.com/docs/concepts/oauth-openid/
    - please feel free to [PR or request more help on the Temporal Web repo](https://github.com/temporalio/web/)

2. You will need to provide a redirect URL to your Oauth Provider. If you are hosting Temporal Web at `http://localhost:8088` (this is configured by `callback_base_uri` in `server/config.yml`), then it is `http://localhost:8088/auth/sso_callback`. 

>     - Temporal Web will be asking for 3 scopes, make sure your provider recognizes these or you may see scope-related errors:

Author: ```suggestion
    - By default, Temporal Web asks for 3 scopes, make sure your provider recognizes these or you may see scope-related errors:
```

>       - `openid` required by some OIDC providers like [auth0](https://auth0.com/docs/scopes/openid-connect-scopes)

Author: maybe just want to collapse the 3 scopes into a single line instead of listing like this, what do you think

Reviewer: if we collapse it would resemble the `scope` param in openid.
On other hand the downside of this approach is if users have too many custom scopes for AuthZ, then having a single line could be harder to read

Author: ok lets leave it for now
